### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -17,11 +16,11 @@ The example includes solutions for both Blazor Server and Blazor WebAssembly hos
 
 Follow the steps below to implement a Theme Switcher in your application:
 
-1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application.
+1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application. The newly created project references the **DevExpress.Blazor** NuGet package that contains DevExpress Blazor UI components and links a package with built-in DevExpress themes.
```
Comment (khamlyuk) [inline Readme.md]: Can we name the theme package name in the parentheses?

File: Readme.md [hunk 2]
```diff
@@ -17,11 +16,11 @@ The example includes solutions for both Blazor Server and Blazor WebAssembly hos
...
+1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application. The newly created project references the **DevExpress.Blazor** NuGet package that contains DevExpress Blazor UI components and links a package with built-in DevExpress themes.
 
 2. Copy this example's [switcher-resources](https://github.com/DevExpress-Examples/blazor-theme-switcher/tree/22.2.3%2B/CS/BlazorServer/switcher/switcher/wwwroot/css/switcher-resources) folder to your application's *wwwroot/css* folder. The *switcher-resources* folder has the following structure:
 
-    * The *themes* folder includes nested folders whose names correspond to Bootstrap theme names. Each nested folder stores a Bootstrap theme's stylesheet (the *bootstrap.min.css* file).
+    * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). Note that the built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package.
```
Comment (khamlyuk) [inline Readme.md]: ...and you do not need to add/link them manually.

File: Readme.md (hunk 1 above)
Comment (SvetlanaMikheeva) [reply Readme.md]:
```suggestion
1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application configured to use DevExpress Blazor components. The newly created project references the **DevExpress.Blazor.Themes** NuGet package with built-in DevExpress themes (as the **DevExpress.Blazor** package's dependency).

```

File: Readme.md (hunk 2 above)
Comment (SvetlanaMikheeva) [reply Readme.md]:
```suggestion
    * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). The built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package that was installed to the application on the previous step.

```

File: Readme.md [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 # How to implement a Theme Switcher in Blazor applications
 
-This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). The choices will include [DevExpress built-in themes](https://github.com/DevExpress/bootstrap-themes) and Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).
+This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). The choices will include [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).
```
Comment (albertov05) [inline Readme.md]: `The choices will include [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).`

What "choices" are you talking about here? For example, are these two choices something you can select in the "switcher" discussed in the preceding sentence? For example, how about:

`This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in the [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). You can switch between [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).`

File: Readme.md [hunk 4]
```diff
@@ -17,11 +16,12 @@ The example includes solutions for both Blazor Server and Blazor WebAssembly hos
...
+1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application configured to use DevExpress Blazor components. The newly created project also references the **DevExpress.Blazor.Themes** NuGet package with built-in DevExpress themes.
 
 2. Copy this example's [switcher-resources](https://github.com/DevExpress-Examples/blazor-theme-switcher/tree/22.2.3%2B/CS/BlazorServer/switcher/switcher/wwwroot/css/switcher-resources) folder to your application's *wwwroot/css* folder. The *switcher-resources* folder has the following structure:
 
-    * The *themes* folder includes nested folders whose names correspond to Bootstrap theme names. Each nested folder stores a Bootstrap theme's stylesheet (the *bootstrap.min.css* file).
+    * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). Note that built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package, which was installed to the application on the previous step.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
    * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). Note that built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package, which was installed to the application in the previous step.
```

File: Readme.md (hunk 3 above)
Comment (SvetlanaMikheeva) [reply Readme.md]:
```suggestion
This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in the [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). You can switch between [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).
```

**Pull request merged**